CHANGELOG.md

```diff
@@ -0,0 +1,154 @@
+# Changelog
+
+## v3.3.0 (2025-10-01)
+
+### Tracer
+* [refactor][breaking] Have 'analyze_plots' return id, not full metadata
+* [refactor][breaking] Rename list_plots_analyses_report_types to list_plots_analysis_report_types
+* [refactor][breaking]  change order of arguments to allow future "url flattening"
+* [api] Add report list, types list, get, precheck, creation and groups and analysis get
+* [refactor] Extract upload and HTTP response check helpers
+* [doc] Fix readme release process section list
+* [refactor] Use new simplified URLs
```

Comment:
I feel there is a question who's the audience for the changelog. If it's end users, then I think `[refactor]` and `[devops]` should be left out ?

Reply:
done